pkg/testing/tools/git/git.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package git
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os/exec"
+	"regexp"
+)
+
+var (
+	releaseBranchRegexp = regexp.MustCompile(`.*(\d+\.\d+)$`)
+)
+
+// GetReleaseBranches returns a list of release branches of the
+// current repository ordered descending by creation date.
+// e.g. 8.13, 8.12, etc.
+func GetReleaseBranches(ctx context.Context) ([]string, error) {
```

Comment:
Why don't we can use a library to avoid launching commands and scanning output meant for humans?
Something like https://github.com/go-git/go-git maybe ?

Reply:
I don't think it's worth to introduce an entire library just for this simple operation. I don't think this code is very complex and it's very optimized to our purpose.